chainercv/visualizations/vis_instance_segmentation.py

```diff
@@ -0,0 +1,98 @@
+from __future__ import division
+
+import numpy as np
+
+from chainercv.visualizations.vis_image import vis_image
+from chainercv.visualizations.vis_semantic_segmentation import _default_cmap
+
+
+def vis_instance_segmentation(
+        img, bbox, mask, label=None, score=None, label_names=None,
+        alpha=1, ax=None):
+    """Visualize instance segmentation.
+
+    Example:
+
+        >>> from chainercv.datasets import SBDInstanceSegmentationDataset
+        >>> from chainercv.datasets \
+        ...     import sbd_instance_segmentation_label_names
+        >>> from chainercv.visualizations import vis_instance_segmentation
+        >>> import matplotlib.pyplot as plot
+        >>> dataset = SBDSegmentationDataset()
+        >>> img, bbox, mask, label = dataset[0]
+        >>> vis_instance_segmentation(
+        ...     img, bbox, mask, label,
+        ...     label_names=sbd_instance_segmentation_label_names,
+        ...     alpha=0.7)
+        >>> plot.show()
+
+    Args:
+        img (~numpy.ndarray): An array of shape :math:`(3, H, W)`.
+            This is in RGB format and the range of its value is
+            :math:`[0, 255]`.
+        bbox (~numpy.ndarray): A float array of shape :math:`(R, 4)`.
+            :math:`R` is the number of objects in the image, and each
+            vector represents a bounding box of an object.
+            The bounding box is :math:`(y_min, x_min, y_max, x_max)`.
+        mask (~numpy.ndarray): A bool array of shape
+            :math`(R, H, W)`.
+            If there is an object, the value of the pixel is :obj:`True`,
+            and otherwise, it is :obj:`False`.
+        label (~numpy.ndarray): An integer array of shape :math:`(R, )`.
+            The values correspond to id for label names stored in
+            :obj:`label_names`.
+        label_names (iterable of strings): Name of labels ordered according
+            to label ids.
+        alpha (float): The value which determines transparency of the figure.
```

Comment:
Is `alpha=1` a reasonable default value?
To me, `alpha=0.7` looks better.
Also, the doc needs to be changed if we are changing this.

Reply:
I prefer alpha=0.7, so I will change it

Reply:
I updated.